hydroflow_lang/src/graph/ops/dest_sink.rs

```diff
@@ -7,11 +7,69 @@ use quote::quote_spanned;
 /// > Arguments: An [async `Sink`](https://docs.rs/futures/latest/futures/sink/trait.Sink.html).
 ///
 /// Consumes items by sending them to an [async `Sink`](https://docs.rs/futures/latest/futures/sink/trait.Sink.html).
-///
-/// This handles a stream of individual items, of an arbitrary type, whereas [`dest_asyncwrite`](#dest_asyncwrite)
-/// handles streams of bytes.
+/// A `Sink` is a thing into which values can be sent, asynchronously. For example, sending items
+/// into a bounded channel.
 ///
 /// Note this operator must be used within a Tokio runtime.
+///
+/// ```rustbook
+/// # #[tokio::main]
+/// # async fn main() {
+/// // In this example we use an _bounded_ channel for our `Sink`. This is for demonstration only,
+/// // instead you should use [`hydroflow::util::unbounded_channel`]. A bounded channel results in
```

Comment:
what's the benefit of the bounded channel in documentation?

Reply:
Added
```
(We can't use
/// // unbounded here since unbounded channels are synchonous to write to and therefore not
/// // `Sink`s.)
```